sample-plugin/pom.xml

```diff
@@ -635,6 +635,40 @@
     </pluginRepositories>
     <build>
         <plugins>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <configuration>
```

Comment:
The configuration has to be duplicated here unfortunately because `sample-plugin` extends from the plugin POM and not the root of the multi-module project. This is basically a hack as explained in https://github.com/jenkinsci/bom/issues/683.